pkg/pipeline/ChartService.go

```diff
@@ -47,6 +48,19 @@ import (
 	"time"
 )
 
+type BulkUpdateInput struct {
+	AppNameIncludes string `json:"appNameIncludes"`
```

Comment:
change
AppNameIncludes ->
Includes:
  AppName
and
AppNameExcludes -> 
Excludes:
  AppName